src/firmata/firmata_mraa.c

```diff
@@ -451,6 +483,8 @@ mraa_firmata_plat_init(const char* uart_dev)
     b->i2c_bus_count = 1;
     b->def_i2c_bus = 0;
     b->i2c_bus[0].bus_id = 0;
+    b->pwm_min_period = 2048000;
+    b->pwm_max_period = 2048000;
```

Comment:
Yes I'm very familiar with the Firmata protocol, but it's unclear that this is related to any constraint of the protocol. 

I asked if it was a typo because this is an incredibly large period in µS to used as a "minimum" (it's over 2 seconds long!). What is the basis/source of this particular value? Based on all other [occurrences](https://github.com/intel-iot-devkit/mraa/search?utf8=%E2%9C%93&q=pwm_max_period) of these properties, a smaller min/max or default would be appropriate.